Assets/StreamingAssets/Quests/$CUREVAM.txt

```diff
@@ -235,9 +235,9 @@ QBN:
 Item _letter_ letter used 1020
 Item _token_ bracelet
 
-Person _vamp1_ face 40 factiontype Vampire_Clan
-Person _vamp2_ face 4 factiontype Vampire_Clan
-Person _hunter_ face 1 factiontype Knightly_Guard anyInfo 1011 rumors 1012
+Person _vamp1_ face 40 factiontype Vampire_Clan remote
+Person _vamp2_ face 4 factiontype Vampire_Clan male remote
```

Comment:
Any reason to make `_vamp2_` male? Message 1023 uses %g and %g2 to refer to them, so I assume they were intended to be either gender.

Reply:
I'm not sure, if it's set as male then it was _probably_ intended, but as always it can be an issue in the original quest. I only fixed those that were obvious flaws.

Reply:
Oh, all those quest files have been regenerated from a tool? I guess I won't review them all, then

Reply:
Yes all quests are updated using a dedicated tool of my own which extracts the necessary fields from QBN data and add them back to DFU in TEMPLATE quest format.